src/constants/accessPermissions.ts

```diff
@@ -52,5 +87,67 @@ export const PAIRINFO_ACCESS: { [chainId in ChainId]: boolean } = {
   [ChainId.SONGBIRD]: false,
   [ChainId.HEDERA_TESTNET]: false,
   [ChainId.NEAR_MAINNET]: false,
-  [ChainId.NEAR_TESTNET]: false
+  [ChainId.NEAR_TESTNET]: false,
+  [ChainId.ETHEREUM]: false,
+  [ChainId.POLYGON]: false,
+  [ChainId.FANTOM]: false,
+  [ChainId.XDAI]: false,
+  [ChainId.BSC]: false,
+  [ChainId.ARBITRUM]: false,
+  [ChainId.CELO]: false,
+  [ChainId.OKXCHAIN]: false,
+  [ChainId.VELAS]: false,
+  [ChainId.AURORA]: false,
+  [ChainId.CRONOS]: false,
+  [ChainId.FUSE]: false,
+  [ChainId.MOONRIVER]: false,
+  [ChainId.MOONBEAM]: false,
+  [ChainId.OP]: false
+}
+
+const onlyBridgeHideOtherMenus: MENU_LINK[] = [
+  MENU_LINK.swap,
+  MENU_LINK.buy,
+  MENU_LINK.pool,
+  MENU_LINK.stake,
+  MENU_LINK.vote,
+  MENU_LINK.migrate,
+  MENU_LINK.airdrop,
+  MENU_LINK.stakev2
+]
+
+const nonEvmHideList: MENU_LINK[] = [
+  MENU_LINK.buy,
+  MENU_LINK.pool,
+  MENU_LINK.stake,
+  MENU_LINK.vote,
+  MENU_LINK.migrate,
+  MENU_LINK.airdrop,
+  MENU_LINK.stakev2
+]
+
+export const HIDE_MENU_ACCESS_MANAGEMENT: { [chainId in ChainId]?: MENU_LINK[] } = {
+  [ChainId.FUJI]: [],
+  [ChainId.AVALANCHE]: [],
+  [ChainId.WAGMI]: [],
+  [ChainId.COSTON]: [],
+  [ChainId.SONGBIRD]: [MENU_LINK.vote],
+  [ChainId.HEDERA_TESTNET]: nonEvmHideList,
```

Comment:
we are considering hedera as evm chain as its compatible with solidity. so here we need 
```suggestion
  [ChainId.HEDERA_TESTNET]: [MENU_LINK.vote, MENU_LINK.migrate, MENU_LINK.airdrop],
```

Reply:
fixed